src/components/Books/Books.jsx

```diff
@@ -18,6 +23,11 @@ const Books = () => {
   );
   return (
     <section className="book-shelf">
+      {loading ? (
+        <BallTriangle />
+      ) : (
+        BookComponent
+      )}
       {BookComponent}
```

Comment:
# thanks for the info 

@Qoosim 

i accidentally create two component while adding loader